tests/us_enrichment/test_Client.js

```diff
@@ -0,0 +1,239 @@
+const chai = require("chai");
+const expect = chai.expect;
+const Client = require("../../src/us_enrichment/Client");
+const Lookup = require("../../src/us_enrichment/Lookup");
+const errors = require("../../src/Errors");
+const MockSender = require("../fixtures/mock_senders").MockSender;
+const MockSenderWithResponse = require("../fixtures/mock_senders").MockSenderWithResponse;
+const Attributes = require("../../src/us_enrichment/Response.js")
```

Comment:
Looks like this can also be commented out

Reply:
Line 5 has been removed on the most recent commit.

Reply:
Removed line 8